tests/acceptance/features/apiSearch/apiSpaceSearch.feature

```diff
@@ -20,7 +20,7 @@ Feature: Search
 
```

Comment:
ops, found one deficiency.
You have switched web to use space dav path, but tests still use only the new DAV path. we need to change it for all tests from apiSpaceSearch.feature like:

```
Scenario Outline: user can find data from the project space
    Given using <dav-path-version> DAV path
    When user "Alice" searches for "*fol*" using the WebDAV API
    Then the HTTP status code should be "207"
    And the search result should contain "4" entries
    And the search result of user "Alice" should contain these entries:
      | /folderMain                                           |
      | /folderMain/SubFolder1                                |
      | /folderMain/SubFolder1/subFOLDER2                     |
      | /folderMain/SubFolder1/subFOLDER2/insideTheFolder.txt |
    Examples:
      | dav-path-version |
      | old              |
      | new              |
    @skipOnRevaMaster
    Examples:
      | dav-path-version |
      | spaces           |
```

Reply:
> @ScharfViktor, not exactly sure what you mean, but if you mean that web uses the 'space/*' endpoint for report requests, ...

yes,  web uses the 'space/*' but api tests uses only new DAV Path. I changed it here https://github.com/owncloud/ocis/commit/109b70eea80a449a2bdad379fee93d937cf842c4 and I'll create PR
after merging your